diff --git a/node_modules/robotjs/src/mouse.c b/node_modules/robotjs/src/mouse.c
index e61e8b7..d2eede1 100644
--- a/node_modules/robotjs/src/mouse.c
+++ b/node_modules/robotjs/src/mouse.c
@@ -342,7 +342,7 @@ void scrollMouse(int x, int y)
 	mouseScrollInputs[1].mi.dwExtraInfo = 0;
 	mouseScrollInputs[1].mi.mouseData = y;

-	SendInput(2, mouseScrollInputs, sizeof(mouseScrollInputs));
+	SendInput(2, mouseScrollInputs, sizeof(INPUT));
 #endif
 }

